.pipelines/azure_pipeline_dev.yaml

```diff
@@ -0,0 +1,76 @@
+# Starter pipeline
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+# Add steps that build, run tests, deploy, and more:
+# https://aka.ms/yaml
+
+trigger:
+- ci_dev
+
+pool:
+  name: Azure-Pipelines-CI-Test-EO
+
+variables:
+  armServiceConnectionName: 'ci-1es-acr-connection'
+  subscription: '9b96ebbd-c57a-42d1-bbe9-b69296e4c7fb'
+  containerRegistry: 'containerinsightsprod'
+  repoImageName: '${{ variables.containerRegistry }}.azurecr.io/public/azuremonitor/containerinsights/cidev'
+
+steps:
+- bash: |
+    commit=$(git rev-parse --short HEAD)
+    echo "##vso[task.setvariable variable=commit;]$commit"
+
+    datetime=$(date +'%Y%m%d%s')
+    echo "##vso[task.setvariable variable=datetime;]$datetime"
+
+    cd deployment/multiarch-agent-deployment/ServiceGroupRoot/Scripts
+    tar -czvf ../artifacts.tar.gz pushAgentToAcr.sh
+
+- task: CopyFiles@2
+  displayName: "Copy ev2 deployment artifacts"
+  inputs:
+    SourceFolder: "$(Build.SourcesDirectory)/deployment"
+    Contents: |
+      **/*
+    TargetFolder: '$(Build.ArtifactStagingDirectory)/build'
+
+- task: CopyFiles@2
+  displayName: "Copy ev2 deployment scripts"
```

Comment:
Once we are ready to use this build for our regular dev and release pipelines, we will also want to copy the yaml files we use to apply the image to test clusters, etc. Reference for what we are copying over now: https://github.com/microsoft/Docker-Provider/blob/ec3d2ef997369fe3a4bb7afbd411ad65f2564286/.pipelines/pipeline.user.linux.official.all_tag.all_phase.all_config.ci_prod.yml#L31

Reply:
Currently the pipelines + builds for multiarch and single plat are different. Once we merge them then the above changes will be added